### PR TITLE
Deleted duplicated example in ternaries.

### PR DESCRIPTION
**Pull request by cn007b**
In ternaries block present same code example: `const maybeNull = value1 > value2 ? 'baz' : null;` as `better` and `best`.
I've deleted this example from `better` example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1899,8 +1899,6 @@ Other Style Guides
       : value1 > value2 ? "baz" : null;
 
     // better
-    const maybeNull = value1 > value2 ? 'baz' : null;
```
Comment (ljharb) [inline README.md]: this isn't actually a separate example, it's defining the code used a few lines down. Maybe it'd be better to just remove the empty lines between `const maybeNull` and `const foo`, in both sections?

Comment (cn007b) [reply README.md]: I'd understood the gist but was perplexed when I've found same code in better and good sections) Yes, it makes sense to delete empty lines, but still, same code will be in 2 sections(
Maybe it's possible to consider something like this:
````
// split into 2 separated ternary expression
const maybeNull = value1 > value2 ? 'baz' : null;

// better
const foo = maybe1 > maybe2
  ? 'bar'
  : maybeNull;

// best
const foo = maybe1 > maybe2 ? 'bar' : maybeNull;
````

Comment (ljharb) [reply README.md]: Sure, that's not bad. (`s/expression/expressions` tho)

**Comment by ljharb** [comment]
Please don't open a second PR; let's reuse this one :-)

**Comment by ljharb** [comment]
Happy to help if you're having issues rebasing!

**Comment by cn007b** [comment]
OK. I've already closed 2nd PR) I don't use rebase in general...
So for me, it's just nightmare

**Comment by ljharb** [comment]
np, i'll take care of it

**Comment by cn007b** [comment]
I saw that you `surgeon with git rebase`)
I've squashed my commits but I was surprised with record `changed the title from`...
and decided that I'd done something wrong...

**Comment by ljharb** [comment]
Next time it comes up, ping me in the gitter channel and I'll help walk you through it.

**Comment by cn007b** [comment]
Ok, thank you so much! Sorry for this mess.

**Pull request merged**